modules/dailymotionBidAdapter.js

```diff
@@ -42,11 +42,15 @@ function getVideoMetadata(bidRequest, bidderRequest) {
     iabcat2: Array.isArray(videoParams.iabcat2)
       ? videoParams.iabcat2
       : Object.keys(iabcat2),
-    id: videoParams.id || '',
-    lang: videoParams.lang || '',
+    id: videoParams.id ||
+      deepAccess(bidderRequest, 'ortb2.site.content.id', ''),
+    lang: videoParams.lang ||
+      deepAccess(bidderRequest, 'ortb2.site.content.language', ''),
```

Comment:
I couldn't found the `content.language` key in documentation, could you share me a link ?

Reply:
In Section 3.2.16
Shares same ISO code as `videoParams.lang`

<img width="509" alt="Capture d’écran 2024-04-19 à 10 57 11" src="https://github.com/dailymotion-oss/Prebid.js/assets/11768029/95180731-0d92-4a65-bf83-c058e4074831">